datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/config.py

```diff
@@ -3,12 +3,15 @@
 # Licensed under a 3-clause BSD style license (see LICENSE)
 import click
 import yaml
+from six import PY2
 
 from datadog_checks.dev.tooling.config_validator.validator import validate_config
 from datadog_checks.dev.tooling.config_validator.validator_errors import SEVERITY_ERROR, SEVERITY_WARNING
+from datadog_checks.dev.tooling.configuration import ConfigSpec
+from datadog_checks.dev.tooling.configuration.consumers import ExampleConsumer
 
-from ....utils import basepath, read_file
-from ...utils import get_config_files, get_valid_checks
+from ....utils import basepath, file_exists, get_parent_dir, path_join, read_file, write_file
+from ...utils import get_config_files, get_config_spec, get_valid_checks, load_manifest
```

Comment:
Four dots really loses the context of where these imports come from.  Any reason they are not imported via `datadog_checks.dev.tooling.utils` and `datadog_checks.dev.tooling.dev.utils`?

Reply:
In general, mostly to avoid loading things exposed at/toward the root. Here's a relevant link: https://github.com/DataDog/integrations-core/pull/3326

Reply:
👍